### DISCUSSION TITLE
save as image

### DISCUSSION THREAD

**Discussion by alinemati45** [lencx/ChatGPT#515] [Q&A]
Hello,
I can not see the option to save as image. how can I fix it?

**Comment by Ethkuil** [comment]
Version? Where did you install this software? Os?

**Reply by alinemati45** [reply]
mac os, Apple M1 Pro. I can share an image to see if you mind. 
thanks,

**Reply by Ethkuil** [reply]
Where did you install this software? The version of your software?(Can be seen at `menu -> ChatGPT -> About`.)

Because it's possible that the version you're using is an unofficial version or an older and the official latest version has fixed it. If not that, then we can confirm that this is likely to be a bug.

**Comment by alinemati45** [comment]
I just download it today.
version: Version 0.11.0 (20230216.160724)

**Reply by Ethkuil** [reply]
> I just download it today.

So where? Do you mean GitHub Releases?

**Reply by alinemati45** [reply]
yes. this address: 

https://github.com/lencx/ChatGPT/releases/download/v0.11.0/ChatGPT_0.11.0_macos_aarch64.dmg

I can install previous version to see if I have same issue?

**Reply by Ethkuil** [reply]
> I can install previous version to see if I have same issue?

Certainly if you would like to. The maintainer will thank you much.

---

The expected UI should be like:

![image](https://user-images.githubusercontent.com/77615248/221466158-4804fa93-57ea-4899-b5e1-6401ba084f8c.png)

When there exists communication, there should be these buttons. For now I think it's likely to be a bug and you can wait for the maintainer lencx to finish further work.

**Reply by alinemati45** [reply]
I know, it is not like it. 

![image](https://user-images.githubusercontent.com/15809162/221467941-2b9c0684-f333-4dbc-a08f-0f447d5c103f.png)

**Reply by alinemati45** [reply]
let me check on the windows computer too. I will keep you posted.

**Reply by alinemati45** [reply]
same thing in the windows. version in windows is 

![image](https://user-images.githubusercontent.com/15809162/221469934-67960946-1b8a-468e-9a06-ca7bb6f85a40.png)

**Reply by Ethkuil** [reply]
I have tried to reproduce the "bug" myself. And I get a funny conclusion in a certain sense. You can try to maximize the windows and see what happens.

**Reply by alinemati45** [reply]
**it works in maximize**. lol. is there any way to fix it?

**Reply by alinemati45** [reply]
the second issue is icon does not show on Menu-bar. I am not sure is my issue but worth to share. Icon will go to behind the camera on mac. 

![image](https://user-images.githubusercontent.com/15809162/221471321-be78f34d-01e1-48ee-96ce-2f27f5e5f75b.png)

**Reply by Ethkuil** [reply]
> **it works in maximize**. lol. is there any way to fix it?

It's a feature not bug. It responds to size change with automatic layout change. You can adjust the size of window to get a suitable layout.

**Reply by Ethkuil** [reply]
> the second issue is icon does not show on Menu-bar. I am not sure is my issue but worth to share. Icon will go to behind the camera on mac.
> 
> ![image](https://user-images.githubusercontent.com/15809162/221471321-be78f34d-01e1-48ee-96ce-2f27f5e5f75b.png)

I have never used mac so am unable to answer. You can try opening a new discussion.

**Comment by lencx** [comment]
> Hello, I can not see the option to save as image. how can I fix it?

The Export button is removed when the window width is less than 768.

**Reply by alinemati45** [reply]
it workds.
thanks,
